module/database/UserDatabase.py

```diff
@@ -33,7 +34,7 @@ def checkAuth(db, user, password):
 
         salt = r[2][:5]
         pw = r[2][5:]
-        h = sha1(salt + password)
+        h = sha1(salt.encode('utf-8') + password.encode('utf-8'))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of insecure MD2, MD4, MD5, or SHA1 hash function.](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=4846282)